share/lib/python/neuron/coreneuron.py

```diff
@@ -160,6 +172,8 @@ def nrncore_arg(self, tstop):
         # args derived from user properties
         if self._gpu:
             arg += " --gpu"
+            if self._num_gpus:
+                arg += " --num-gpus %d" % self._num_gpus
```

Comment:
In order to have this tested, shouldn't this be set by some test like test_direct.py ?